README.md

```diff
@@ -197,3 +197,29 @@ Questions?
 ----------
 For help and support with pyGSTi, please contact the authors at
 pygsti@sandia.gov.
+
```

Comment:
I forgot I added this, I think I did so in response to a request from @dhothem. We should probably also include the citation for Zenodo.

Reply:
That might be better. I'd want to double-check how DOE Code is grabbing the author list, though. If it is through our internal CA process it might not be as flexible as we'd like.